edx_proctoring/templates/proctoring/seq_proctored_exam_ready_to_submit.html

```diff
@@ -0,0 +1,62 @@
+{% load i18n %}
+<div class="sequence proctored-exam completed" data-exam-id="{{exam_id}}">
```

Comment:
I think this template is now going to serve as the confirmation page after a student clicks "End my exam" in the timer bar. 
@griffresch @marcotuts @chrisndodge Please take a look: do my suggestions for text for this template make sense in this context?